rpcs3/Emu/HDD/HDD.cpp

```diff
@@ -19,7 +19,7 @@ void vfsHDDManager::CreateEntry(vfsHDD_Entry& entry)
 	CreateBlock(entry);
 }
 
```

Comment:
@vlj Sounds like it would make sense. I'll also make some return codes for the virtual memory helper functions.